Components/Movielist.js

```diff
@@ -0,0 +1,129 @@
+const list = [
```

Comment:
.json and .js files are duplicate. Why do you need both?

Reply:
I was trying some different method. I have deleted .json file.